packages/fluentui/docs/src/prototypes/VirtualizedStickyTree/VirtualizedStickyTree.tsx

```diff
@@ -0,0 +1,486 @@
+import * as React from 'react';
+import { VariableSizeList as List } from 'react-window';
+import {
+  ComponentEventHandler,
+  ObjectShorthandCollection,
+  Tree,
+  TreeItemProps,
+  treeTitleClassName,
+  teamsTheme,
+  Accessibility,
+  TreeBehaviorProps,
+  AccessibilityAttributes,
+} from '@fluentui/react-northstar';
+import StickyTreeTitle from './StickyTreeTitle';
+import { InnerElementContext, InnerElementContextType } from './context';
+
+class ItemWrapper extends React.PureComponent<{
+  index: number;
+  isScrolling?: boolean;
+  style: Object;
+  data: any;
+}> {
+  render() {
+    const { index, style, data } = this.props;
+    const item = data.renderedItems[index];
+    if (item) {
+      return data.isNonSticky(item)
+        ? React.cloneElement(item, {
+            style,
+            onTitleClick: data.onClick,
+            onFocusParent: data.handleFocusParent,
+            onFocusFirstChild: data.handleFocusFirstChild,
+            onSiblingsExpand: (event: React.SyntheticEvent<HTMLElement>, treeItemProps: TreeItemProps) => {
+              if (treeItemProps.level !== 1)
```

Comment:
Always wrap block statements by curly braces
@layershifter - can we check why lint did not explode? why prettier did not fix that?